test/e2e/tests/test_table.py

```diff
@@ -152,19 +132,40 @@ def test_table_update_tags(self, dynamodb_client):
 
         # Patch k8s resource
         k8s.patch_custom_resource(ref, cr)
-        time.sleep(UPDATE_TAGS_WAIT_AFTER_SECONDS)
+        time.sleep(MODIFY_WAIT_AFTER_SECONDS)
 
         table_tags = tag.clean(get_resource_tags(cr["status"]["ackResourceMetadata"]["arn"]))
         assert len(table_tags) == len(tags)
         assert table_tags[0]['Key'] == tags[0]['key']
         assert table_tags[0]['Value'] == tags[0]['value']
 
-        # Delete k8s resource
-        _, deleted = k8s.delete_custom_resource(ref)
-        assert deleted is True
+    def test_enable_ttl(self, dynamodb_client, forum_table):
```

Comment:
no test for disabling again after enabling?

Reply:
There is a limitation on this API. You are only allowed to update this field once every hour. I have verified manually that disabling it works, so that's the best I can do for automated tests